core/server/api/canary/newsletters.js

```diff
@@ -17,6 +18,13 @@ module.exports = {
             'order',
```

Comment:
You're missing include in the permitted options

Reply:
yes that's what I've found too, thanks for looking into that 🙏